framework/py/flwr/server/grid/inmemory_grid_test.py

```diff
@@ -46,7 +47,9 @@ def push_messages(grid: InMemoryGrid, num_nodes: int) -> tuple[Iterable[str], in
     """Help push messages to state."""
     for _ in range(num_nodes):
         node_id = grid.state.create_node(
-            secrets.token_bytes(32), heartbeat_interval=HEARTBEAT_MAX_INTERVAL
+            NOOP_FLWR_AID,
```

Comment:
in the other tests files we don't use `NOOP_FLWR_AID`, instead we set some ad-hoc ids `mock_flwr_aid` and `mock_owner`. Should we use `NOOP_FLWR_AID` throughout ?